UPGRADING.md

```diff
@@ -3,6 +3,9 @@ Upgrading Grape
 
 ### Upgrading to >= 2.1.0
 
+#### Changes in rescue_from
+`rack_response` has been deprecated and `error_response` has been removed. Use `error!` instead.
```

Comment:
The `rack_response` method ... (just to make it a sentence)

Link to the PR here and a couple missing below.

```
See [#2414](https://github.com/ruby-grape/grape/pull/2414) for more information.
```